boilerplate/_lib/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
```

Comment:
We use this as a base to build go and then use `registry.access.redhat.com/ubi8/ubi-minimal:latest` on top for operators and CI containers. Can we do that here it may also help with packages below that I assume are missing from yum?

Reply:
It is possible but I don't think it's necessary. Also it will complicate stuff because we will have to copy the git binaries from the first to second stage. So stuff like shell autocompletion may not work. Also it's probably better to have a full featured image to run the boilerplate stuff esp. for stuff like debugging. And since this will never be run in production it doesn't pose a security risk. The only problem is size of the image.

Reply:
No objection here, but "shell autocompletion" caught my eye. Under what circumstances is that relevant? This image is for running containerized automated builds & checks, and for running the operator, right? Should be non-interactive except for the 0.01% of the time we need to jump in and debug something?

Reply:
Shell autocompletion is not the only thing. I'm no C++ expert but the way I understand it when a C++ binary is built it is linked to libraries on the machine. Simply copying the binary may not work like in the case of Go.